### ISSUE TITLE
TF2 segfaults on start

### ISSUE THREAD

**Issue by netfab** [ValveSoftware/Source-1-Games#2520]
I did not played TF2 for months. 

Just give it a new try, and TF2 is segfaulting on start \:
> Team Fortress 2/hl2.sh : line 72 :  6403 Segmentation fault  ${GAME_DEBUGGER} "${GAMEROOT}"/${GAMEEXE} "$@"

OS \: Gentoo
Steam beta \: no
Drivers \: nvidia-drivers-390.25
System infos \: [here](https://gist.github.com/netfab/077a29e2b7a66da899cfd1efaa9b07ae)
[crash_dump.zip](https://github.com/ValveSoftware/Source-1-Games/files/1713518/crash_dump.zip)

Can someone please have a look ? I do not have any problems with other games. Thanks.

**Comment by kisak-valve** [comment]
Hello @netfab, does `/tmp/dumps/<username>_stdout.txt` have a hint as to what's going on? Do you have the `steamgames_tf2` useflag enabled for games-util/steam-games-meta?

For reference, the attached minidump is a DUMP_REQUESTED in crashhandler.so

**Comment by netfab** [comment]
Before the segfault I have some errors concerning gameoverlayrenderer.so LD_PRELOAD \: [stdout](https://gist.github.com/netfab/bebbdffd95c4121c94b42ae27c4c25da)

> Do you have the steamgames_tf2 useflag enabled for games-util/steam-games-meta?

No, but is it really required since I'm using the steamruntime package ?

**Comment by netfab** [comment]
Hi,

After each TF2 update I'm trying it, but it always crash in the same way.
Since the first post I updated some packages :
 - nvidia-drivers 396.54
 - kernel 4.14.65

I also tried various things, including :
 - deleted all game files, and re-downloaded it
 - -nojoy -nointro -autoconfig -windowed command line options
 - mat_queue_mode "0" or "1"

No result.

I also tried to run it from command line with :
> GAME_DEBUGGER="strace -f -o strace.log" ./hl2.sh

The strace.log ends with segfault after a [futex call](https://gist.github.com/netfab/6ca4304a18ecd8d285278561aa8c5a66).
